EventFlow.Redis/ReadStore/Extensions.cs

```diff
@@ -0,0 +1,11 @@
+﻿using StackExchange.Redis;
+
+namespace EventFlow.Redis.ReadStore;
+
+public static class Extensions
```

Comment:
Please see if you can mark this as internal as it will "pollute" the `Dictionary<string, string>` extensions for anyone using EventFlow.